src/otp2/controller.ts

```diff
@@ -107,7 +107,6 @@ function getTripPatternsVariables(params: any): any {
         modes = DEFAULT_MODES,
         transportSubmodes = [],
         wheelchairAccessible = false,
-        limit = 5,
```

Comment:
Sjekk at Sdk ikke har en default limit

Reply:
SDK har ein default limit, men me bruker ikkje SDKen for OTP2.